app/components/Home.tsx

```diff
@@ -0,0 +1,270 @@
+import * as React from 'react';
+const { EditorKit, EditorKitDelegate } = require('sn-editor-kit');
+const MarkdownIt = require('markdown-it');
+
+const EditMode = 0;
+const SplitMode = 1;
+const PreviewMode = 2;
+
+const modes = [
+  { mode: EditMode, label: "Edit", css: "edit" },
+  { mode: SplitMode, label: "Split", css: "split" },
+  { mode: PreviewMode, label: "Preview", css: "preview" },
+];
+
+const markdownitOptions = {
+  // automatically render raw links as anchors.
+  linkify: true
+};
+
+const MarkdownParser = MarkdownIt(markdownitOptions)
+  .use(require('markdown-it-footnote'))
+  .use(require('markdown-it-task-lists'))
+  .use(require('markdown-it-highlightjs'));
+
+type AppProps = {
+  text?: string,
+  mode: any;
+  platform?: any,
```

Comment:
`any` should be a last resort only. In this case we should either break these into enums or string options.

For example if platform can be either `desktop` or `mobile`, the type can be `platform: 'desktop' | 'mobile'`

If you use `any` you essentially miss out on TS benefits.

Reply:
OK, I updated `platform` and split mode into three different states: https://github.com/sn-extensions/markdown-basic/pull/29/commits/d0de1380b900e7ebee726cf0f8e133e70e7a2d9b. What do you think of that?